packages/language-server/src/nativeTypes.ts

```diff
@@ -0,0 +1,29 @@
+import exec from './exec'
+
+export interface NativeTypeConstructors {
+    name: string,
+    _number_of_args: number,
+    _number_of_optional_args: number,
+    prisma_type: string,
+}
+
+export default async function nativeTypeConstructors(
+  execPath: string,
+  text: string,
+  onError?: (errorMessage: string) => void,
+): Promise<NativeTypeConstructors[]> {
+  try {
+    const result = await exec(execPath,  ['native-types'], text)
```

Comment:
Due to this `await`, many functions had to be made async. Is there a strong reason for that vs using `execSync` here. I don't think native types would change a lot right, it is still Prisma defined and used by the user and not user-defined?

Reply:
We can just use execa.sync() here, that will make this a lot simpler 👍

Reply:
There still needs to be an any in the main autocompletion function, since I need to call util.getBinPath(), which is async, to get the binary path.